src/components/DsfrInput/DsfrInputGroup.vue

```diff
@@ -57,7 +57,9 @@ export default defineComponent({
       return this.errorMessage || this.validMessage
     },
     messageClass () {
-      return this.errorMessage ? 'fr-message--error' : 'fr-message--valid'
+      return this.errorMessage
```

Comment:
Ces classes ont changé, effectivement.

`return this.errorMessage ? 'fr-error-text' : 'fr-valid-text'`  suffira